phpcs.xml.dist

```diff
@@ -24,6 +24,7 @@
         <exclude name="SlevomatCodingStandard.TypeHints.TypeHintDeclaration.MissingTraversableReturnTypeHintSpecification" />
         <exclude name="SlevomatCodingStandard.TypeHints.ParameterTypeHint.MissingNativeTypeHint" />
         <exclude name="SlevomatCodingStandard.TypeHints.TypeHintDeclaration.MissingParameterTypeHint" />
+        <exclude name="SlevomatCodingStandard.Namespaces.UnusedUses" />
```

Comment:
Did you exclude this intentionally? I kinda like this one but I assume you had a good reason. If it was because of a single file, you can make your excludes limit itself to one file.

Reply:
I'm talking about it here:
https://github.com/phpDocumentor/phpDocumentor/pull/2239#issuecomment-579870267
https://github.com/phpDocumentor/phpDocumentor/pull/2239#issuecomment-579945218

tdlr: we use an old version of slevomat which can't read generics notation yet. This could be added back if we switch to Doctrine-standard ^5

Reply:
:flushed: You are right, I did not read your comments carefully enough

Reply:
Commented on https://github.com/phpDocumentor/phpDocumentor/pull/2243 if we can include this